src/app/utils/ValidatorUtils.ts

```diff
@@ -370,13 +364,28 @@ export class ValidatorUtils extends CacheModule {
 		}
 
 		const result = request.parse(response)[0]
-		this.lastFreshTime = Date.now()
 		this.currentEpoch = result.currentEpoch[0]
 		this.olderEpoch = result.olderEpoch[0]
 		this.rocketpoolStats = result.rocketpool_network_stats[0]
 		const validatorEffectivenessResponse = result.effectiveness
 		const validatorsResponse = result.validators
 
+		this.syncCommitteesStats = null
+		const hideSyncCommitteesStats = true // wait for the explorer to show sync committee stats too before showing them in the app
+		if (result.sync_committees_stats && !hideSyncCommitteesStats) {
```

Comment:
Please move calculations to the OverviewController

Reply:
Good suggestion, done.